File: digitalpassbook2/app/src/main/java/com/example/digitalpassbook/Student.java
package com.example.digitalpassbook;
import com.google.gson.annotations.SerializedName;

public class Student {
    @SerializedName("id")
    private int id;

    @SerializedName("name")
    private String name;

    @SerializedName("netId")
    private String netId;

    public Student(String fullName, String netID) {
        this.name = fullName;
        this.netId = netID;
    }

    public int getId() { return this.id; }
    public String getName() { return this.name; }
    public String getNetid() { return this.netId; }
}

File: digitalpassbook2/app/src/main/java/com/example/digitalpassbook2/Student.kt
package com.example.digitalpassbook2

import com.google.gson.annotations.SerializedName

class Student(
    @field:SerializedName("name") val name: String,
    @field:SerializedName("netID") val netid: String
) {
    @SerializedName("id")
    val id = 0

}